test/pylib/start_mock.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+import asyncio
+import sys
+import signal
+import argparse
+from s3_server_mock import MockS3Server
+
+
+async def run():
+    parser = argparse.ArgumentParser(description="Start a S3 mock server")
+    parser.add_argument('--host', default='127.0.0.1')
+    parser.add_argument('--port', default='2012')
+    args = parser.parse_args()
+    server = MockS3Server(args.host, int(args.port))
+
+    print('Starting S3 mock server')
+    await server.start()
+    signal.sigwait({signal.SIGINT,signal.SIGTERM})
```

Comment:
nit, might want to add a space after `,`.

Reply:
Done